### ISSUE TITLE
Choco push timeouts

### ISSUE THREAD

**Issue by Thilas** [chocolatey/choco#1726]
### What You Are Seeing?
From time to time, choco push timeouts with the following error even though the push succeeded:
```
Failed to process request. 'Origin Time-out'.  The remote server returned an error: (524) Origin Time-out..
```

@AGSPhoenix mentioned a similar error on #1285.

### What is Expected?
Not to have the timeout error.

### How Did You Get This To Happen? (Steps to Reproduce)
I'm using AU and the issue happens randomly.

### Output Log
AU log: https://ci.appveyor.com/project/Thilas/chocolatey-packages/build/3232

In this case, the package to push was really small.

**Comment by ferventcoder** [comment]
@Thilas yes, something related to chocolatey.org. I'm going to move this issue over.
